src/applications/static-pages/cta-widget/components/messages/mvi/NotFound.jsx

```diff
@@ -20,10 +17,9 @@ const NotFound = () => {
         <p>You can verify your identity in one of these 2 ways:</p>
         <h5>Call the VA benefits hotline</h5>
         <p>
-          Please call us at <a href="tel:800-827-1000">800-827-1000</a>. We’re
-          here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. If you have
-          hearing loss, call TTY:{' '}
-          <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />.
+          Please call us at <va-telephone contact="8008271000" />. We’re Monday
+          through Friday, 8:00 a.m. to 9:00 p.m. ET. If you have hearing hearing
+          loss, call TTY: <va-telephone contact={CONTACTS['711']} />.
```

Comment:
You could remove the "TTY:" from your text blocks and use the `tty` prop on the `va-telephone` to help with any TTY references in this PR.